snap/hooks/install

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+#
+# Copyright 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+snapctl stop --disable ${SNAP_NAME}
```

Comment:
Do we really need this? It's necessary for the dashboard as it's deployed via a charm but not here. Lets start the service with a minimal config right away

Reply:
If the service is a systemd daemon notified-based one, we need this as people need to manually connect the `daemon-notify` plug as the snap got installed, otherwise we can make it as a simple daemon as you mentioned above.

Reply:
Ok, we will have to ask on the snapcraft forum for an update of the snap declaration to allow the plug automatically at installation time.